src/nvim/api/vim.c

```diff
@@ -44,6 +45,45 @@
 # include "api/vim.c.generated.h"
 #endif
 
+
+/// Create a fold from line "start" to line "end" (inclusive) in the current
+/// window.
+/// @param window
+/// @param start starting line
+/// @param end starting end
+/// @param err
+void nvim_fold_create(Window window, Integer start, Integer end, Error *err)
```

Comment:
Should these be in `api/window.c` ?

see `:help dev-api` for API naming conventions: `nvim_win_set_fold()`,  `nvim_win_del_fold()`.